guillotina/content.py

```diff
@@ -694,26 +696,36 @@ def get_all_behavior_interfaces(content) -> list:
 
     from guillotina.content import create_content_in_container
 
+    creators = context.creators
+    contributors = context.contributors
+    user_id = get_authenticated_user_id()
+    if reset_acl:
+        creators = [user_id]
+        contributors = [user_id]
     new_obj = await create_content_in_container(
         destination_ob,
         context.type_name,
         new_id,
         id=new_id,
-        creators=context.creators,
-        contributors=context.contributors,
+        creators=creators,
+        contributors=contributors,
         check_security=check_permission,
     )
-
     for key in context.__dict__.keys():
         if key.startswith("__") or key.startswith("_BaseObject"):
             continue
-        if key in ("id",):
+        if key in ("id", "creators", "contributors"):
             continue
         new_obj.__dict__[key] = context.__dict__[key]
+
     new_obj.__acl__ = context.__acl__
+    if reset_acl:
+        prinrole = IPrincipalRoleManager(context)
+        prinrole.map._clear()
```

Comment:
Should do `new_obj.__acl__ = None` instead!

Reply:
ok, thanks